src/System.AppContext/src/ApiCompatBaseline.net462.txt

```diff
@@ -0,0 +1 @@
+MembersMustExist : Member 'System.AppContext.TargetFrameworkName.get()' does not exist in the implementation but it does exist in the contract.
```

Comment:
This looks like it is resulting from the fact that we don't have a real net462 targeting pack. @venkat-raman251 please file a bug to get that updated.